e2e/cypress/integration/toast/toast_spec.js

```diff
@@ -263,6 +263,45 @@ describe('toasts', () => {
             cy.get('div.toast').should('be.visible').contains('Viewing message history');
         });
     });
+
+    it('MM-T1785 Toast - When marking post as unread', () => {
+        visitTownSquareAndWaitForPageToLoad();
+
+        // # Add 40 posts to create enough space from bottom for making channel scrollable
+        for (let index = 0; index < 40; index++) {
```

Comment:
I'm good as is but can we reduce this to maybe 30? My only concern is about the longer duration it's taking to post 10 more posts.

Reply:
Yes, I have checked it and we are getting scrollable channel in 30 posts itself.